lib/task/import/validate/csvColumnNameValidator.class.php

```diff
@@ -123,29 +123,34 @@ protected function testColumnName(string $columnName)
     protected function loadObjectColumnNames($resourceClass)
     {
         $resourceTypeBaseConfigFile = $resourceClass.'.yml';
-        $config = QubitFlatfileExport::loadResourceConfigFile($resourceTypeBaseConfigFile, 'base');
 
-        $this->validColumnNames = $config['columnNames'];
-        $standardColumns = isset($config['direct']) ? $config['direct'] : [];
-        $columnMap = isset($config['map']) ? $config['map'] : [];
-        $propertyMap = isset($config['property']) ? $config['property'] : [];
+        try {
+            $config = QubitFlatfileExport::loadResourceConfigFile($resourceTypeBaseConfigFile, 'base');
 
-        // If column names/order aren't specified, derive them
-        if (null === $this->validColumnNames) {
-            // Add standard columns
-            $this->validColumnNames = (null !== $standardColumns) ? $standardColumns : [];
+            $this->validColumnNames = $config['columnNames'];
+            $standardColumns = isset($config['direct']) ? $config['direct'] : [];
+            $columnMap = isset($config['map']) ? $config['map'] : [];
+            $propertyMap = isset($config['property']) ? $config['property'] : [];
 
-            // Add from column map
-            if (null !== $columnMap) {
-                $this->validColumnNames = array_merge($this->validColumnNames, array_values($columnMap));
-            }
+            // If column names/order aren't specified, derive them
+            if (null === $this->validColumnNames) {
+                // Add standard columns
+                $this->validColumnNames = (null !== $standardColumns) ? $standardColumns : [];
+
+                // Add from column map
+                if (null !== $columnMap) {
+                    $this->validColumnNames = array_merge($this->validColumnNames, array_values($columnMap));
+                }
 
-            // Add from property map
-            if (null !== $propertyMap) {
-                $this->validColumnNames = array_merge($this->validColumnNames, array_values($propertyMap));
+                // Add from property map
+                if (null !== $propertyMap) {
+                    $this->validColumnNames = array_merge($this->validColumnNames, array_values($propertyMap));
+                }
             }
-        }
 
-        $this->validColumnNamesLowercase = array_map('strtolower', $this->validColumnNames);
+            $this->validColumnNamesLowercase = array_map('strtolower', $this->validColumnNames);
+        } catch (Exception $e) {
```

Comment:
If this is supposed to catch exceptions from `QubitFlatfileExport::loadResourceConfigFile()` then I think it's better to put the catch right after line 128 to avoid catching unrelated exceptions from the rest of the intervening code.

Reply:
Agreed - will change.